lib/knapsack_pro/runners/queue/rspec_runner.rb

```diff
@@ -151,6 +157,22 @@ def self.rspec_clear_examples
             ::RSpec.configuration.reset_filters
           end
         end
+
+        def self.args_with_seed_option_added_when_viable(args, rspec_runner)
+          order_option = adapter_class.order_option(args)
+
+          if order_option
+            # Don't add the seed option for order other than random, e.g. `defined`
+            return args unless order_option include?('rand')
```

Comment:
1. should there be a dot?
`order_option.include?('rand')`

Are you calling the `include?`  method on the `order_option` variable, right?

2. I'm not sure, is this working with both options? `rand` and `random`?

From RSpec docs `$ rspec --help`:

```
  --order TYPE[:SEED]            Run examples by the specified order type.
                                         [defined]           examples and groups are run in the order they are defined
                                         [rand]              randomize the order of groups and examples
                                         [random]            alias for rand
                                         [random:SEED]       e.g. --order random:123
                                         [recently-modified] run the most recently modified files first
```

Reply:
Regarding 2. I see you already tested that in the `spec/knapsack_pro/adapters/rspec_adapter_spec.rb` file. Please ignore this question.

Reply:
> 1. should there be a dot?

Obviously, yes. Thanks for catching this.